lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
You don't have access to selector string outside of the this.elements return scope. 

Adding an if introduces logic and doesn't give you anything in return.